saleor/plugins/anonymize/plugin.py

```diff
@@ -12,6 +12,7 @@ class AnonymizePlugin(BasePlugin):
     """Anonymize all user data in the checkout, user profile and its orders."""
 
     PLUGIN_NAME = "Anonymize"
+    PLUGIN_ID = "mirumee.taxes.anonymize"
```

Comment:
This one is not related to taxes, sorry if I suggested earlier to rename it like that. No idea how to name it, maybe `mirumee.checkout.anonymize`?

Reply:
Or just `mirumee.anonymize`.